cogboard-app/src/main/kotlin/com/cognifide/cogboard/logStorage/LogStorage.kt

```diff
@@ -281,7 +279,7 @@ class LogStorage(
         }
 
         /** Returns a database for storing logs and collection configurations. */
-        private val database: MongoDatabase?
+        val database: MongoDatabase?
```

Comment:
dlaczego nie private?

Reply:
Ponieważ to pole jest używane również przez `LogController`.